lib/ruled/client.lua

```diff
@@ -694,7 +694,7 @@ return setmetatable(module, {
                 crules:append_rules("awful.rules", v)
             end
         else
-            rawset(k, v)
+            rawset(module, k, v)
```

Comment:
Might be worth covering this with a test now?

Let me know and I can look into this myself.

Reply:
i was thinking the same from the same glance, until noticed what it's deprecated in 5.x anyway